docs/BUILD

```diff
@@ -20,53 +20,26 @@ load("@aspect_bazel_lib//lib:copy_to_directory.bzl", "copy_to_directory")
 
 package(default_visibility = ["//visibility:public"])
 
+# Build all documentation as a unified Sphinx project
 filegroup(
-    name = "root_docs_sources",
```

Comment:
@RyaliNvidia Can you take a look at the changes here to make sure I didn't mess anything up?